docs/user-guide/http-api.rst

```diff
@@ -25,7 +25,7 @@ Returns the currently loaded settings. For specific settings please see :ref:`th
 
 Example Call:
 
-.. code-block::
+.. code-block:: shell
```

Comment:
maybe use `console` instead of `shell` for consistency

Reply:
You are right. Will be fixed too with the next force push.